### ISSUE TITLE
Trying to run V4.5 after Windows update ruined my V3

### ISSUE THREAD

**Issue by juisex** [lllyasviel/style2paints#145]
OK, here we go,
Since the last update for Windows 10 (2004) my V3 style2paints got *****
So I decided to format my computer and try to use V4.5
The guide says "only download and double click as a video game"
I'm getting unable to run the V4.5, brand new Windows 10 installation hardware setting:
CPU: Xeon 1220 v3
GPU: GTX 1060 6GB
RAM:  8GB DDR 3

Syslog:
Traceback (most recent call last):
  File "Style2PaintsV45.py", line 595, in <module>
  File "Style2PaintsV45.py", line 33, in main_exec
  File "g:\s2p_env\env_s2p\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
  File "site-packages\cv2\__init__.py", line 3, in <module>
ImportError: DLL load failed: The specified module could not be found.

**Comment by lllyasviel** [comment]
Are you using 64bit windows 10 or 32bit windows 10? It seems that "DLL load failed" error happens in 32bit systems.

**Comment by juisex** [comment]
64 bits, after the post I've downgraded my Windows 10 to 1903v2 version, the V3 is working properly and the V4.5 either.
If I would recommend a version for stability would be Windows 10 1903v2

**Comment by gceboh** [comment]
Possible Reason:

1. Several months ago, I remembered 360 AntiVirus Software's **Cloud Scanning Engine** report some `.pyd` files in Style2Paints V4.5 as virus. And auto delete the file. (I can't remember the filename clealy, perhaps it is `assets\cv2\cv2.cp36-win_amd64.pyd`)

**I know it is FALSE POSITIVE!** and restore the file. It is not the first time it report some development files/library as virus.

Infact, **it is a bug of the Cloud Scanning Engine**. I can't reproduce it under [virus total](http://www.virustotal.com). After several days, the Cloud Scanning Engine bug seems to be fixed. I scan it again and the result is ok without virus.

Cloud Scanning Engine may be based on big data, and the executable files that owned by very few persons may be judged as suspicious.

AntiVirus Software's Cloud Scanning Engine is continuously updating, and its bug is hard to reproduce.

2. By the way, I noticed that 360 AntiVirus Software sometimes has a **bug** that it will **auto detete the detected files without any popup message**. The files are found in Antivirus Quarantine. I'm not sure this bug has been fixed now.

**Comment by gceboh** [comment]
update:
auto detete the detected files (**or set the file as "access denied"**)  without any popup message.

**Comment by gceboh** [comment]
Some strange bugs I encountered with 360 AntiVirus earlier (I'm not sure they have been fixed):
(1) Occasionally, the exclusion list doesn't work.
(2) Occasionally, "temporarily disable the real-time protect" can't prevent it from setting files as "access denied". Only completely EXIT can unlock the detected files.

**So, I suggest completely CLOSE the antivirus software, before debugging some strange bugs of Style2Paints.**
